content/webapp/components/Body/Body.tsx

```diff
@@ -486,12 +483,6 @@ const Body: FunctionComponent<Props> = ({
                     />
                   </LayoutWidth>
                 )}
-                {/* deprecated */}
-                {slice.type === 'deprecatedImageList' && (
```

Comment:
I grabbed a local copy of our Prismic data – we no longer use this slice anywhere, so I just removed it rather than making it compile properly.

There's a single use of the MediaObjectList slice, or I'd have removed that too.